blogposts/announcing-sourcegraph-3.14.md

```diff
@@ -0,0 +1,252 @@
+
+[**🐳 Deploy and scale Sourcegraph with Docker Compose**](#deploy-and-scale-sourcegraph-with-docker-compose)<br />
+
+[**📣 Improved campaign creation, management, and syncing performance**](#improved-campaign-creation-management-and-syncing-performance)<br />
+
+[**⚠️ Required migration for Kubernetes deployments**](#required-migration-for-kubernetes-deployments)<br />
+Sourcegraph is easier to deploy in environments with strict security requirements.
+
+[**📈 Easier debugging of Kubernetes deployments with profiling and tracing**](#easier-debugging-of-kubernetes-deployments-with-profiling-and-tracing)<br />
+
+[**🚨 Search insights and alerts added to admin monitoring**](#search-insights-and-alerts-added-to-admin-monitoring)<br />
+
+[**🧮 Estimate required resources for your Sourcegraph instance**](#estimate-required-resources-for-your-sourcegraph-instance)<br />
+
+[**🙅‍♀️ Option to exclude all forks/archives from syncing to Sourcegraph for GitHub**](#option-to-exclude-all-forksarchives-from-syncing-to-sourcegraph-for-github)<br />
+
+[**🧠 Go LSIF indexer is 3x faster**](#go-lsif-indexer-is-3x-faster)<br />
+
+[**🛎 Aggregated search data added to Sourcegraph pings**](#aggregated-search-data-added-to-sourcegraph-pings)<br />
+
+[**🔎 Experimental smart search bar**](#experimental-smart-search-bar)<br />
+
+[**📝 Changelog**](#changelog)<br />
+Every detail that changed in this release
+
+[**🎖️ Thank you**](#thank-you)<br />
+Sourcegraph couldn't be what it is without the community
+
+</div>
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
+
+## Faster syncing of GitLab and Bitbucket Server repository permissions
+
+TODO: Video?
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Sourcegraph permissions syncing is now more sophisticated to better handle repository permissions at scale. Rather than updating a user’s permissions when they log in and potentially blocking them from seeing search results, Sourcegraph now syncs these permissions opportunistically, as part of a background service. Users are now able to see results from their search queries immediately. Currently, this improvement is supported for GitLab and Bitbucket Server, with GitHub and other code hosts coming soon. See the [documentation for more details](#).
```

Comment:
```suggestion
Sourcegraph permissions syncing is now more sophisticated to better handle repository permissions at scale. Rather than updating a user’s permissions when they log in and potentially blocking them from seeing search results, Sourcegraph now syncs these permissions opportunistically, as part of a background service. Users are now able to see results from their search queries immediately. Currently, this improvement is supported for GitLab and Bitbucket Server, with GitHub and other code hosts coming soon. See the [documentation for more details](https://docs.sourcegraph.com/admin/repo/permissions#background-permissions-syncing).
```

Reply:
Added the real link.

Reply:
Thanks @unknwon !